src/components/validation/mod.rs

```diff
@@ -421,8 +421,7 @@ fn get_validation_configuration_from_test_case_path(
 }
 
 #[cfg(feature = "component-validation-runner")]
-pub fn validate_component(test_case_data_path: &str) {
-    let test_case_data_path = std::path::PathBuf::from(test_case_data_path.to_string());
+pub fn validate_component(test_case_data_path: std::path::PathBuf) {
```

Comment:
Does this need to be a `PathBuf` or can it just be a `&Path`?

Reply:
It looks like the reason this is a `PathBuf` is because it is stored in the `Runner` struct and `Path` doesn't own the string of the path data.

I'm of the mind to not mess with it at this time because it's not causing problems, unless you feel strongly about it.